ts/features/wallet/onboarding/bancomat/navigation/routes.ts

```diff
@@ -1,7 +1,8 @@
 const WALLET_ONBOARDING_BANCOMAT_ROUTES = {
   MAIN: "WALLET_ONBOARDING_BANCOMAT_MAIN",
 
-  CHOOSE_BANK: "WALLET_ONBOARDING_BANCOMAT_CHOOSE_BANK",
+  CHOOSE_BANK_INFO: "WALLET_ONBOARDING_BANCOMAT_CHOOSE_BANK_INFO",
```

Comment:
What you think about renaming this route? We have moved the choose of the bank in "WALLET_ONBOARDING_CHOOSE_BANK_SCREEN". This could became `  BANCOMAT_START: "WALLET_ONBOARDING_BANCOMAT_START", ` or something similar, in order to explain that is the first screen of the workflow, what you think?